dashboard/apps/dashboard/spreadsheets.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+import re
+
+from openpyxl.styles import Style, Font
+from openpyxl.workbook import Workbook
+
+from dashboard.libs.date_tools import parse_date
+
+
+class Products:
+
+    date_style = Style(number_format='DD/MM/YYYY')
+    bold_font = Font(bold=True)
+    bold_style = Style(font=bold_font)
+    currency_style = Style(number_format='£#,##0.00')
+
+    def __init__(self, products):
+        self.workbook = Workbook()
+        self.fill_main_sheet(self.workbook.active, products)
+        if len(products) == 1:
+            # only create monthly spend sheet when there is one product
+            self.fill_monthly_spend_sheet(self.workbook.create_sheet(), products[0])
```

Comment:
the comment should be one line above. i shall add the reason why to the comment as well, since it does feel strange without explaining the context. the main reason is there is not user need for this monthly spend sheet for all products. from a programming point of view, there will be more effort to align the months for different products, e.g. align the columns of a product with lifetime of 2014-2016 with another product 2015-2017.